src/main/webapp/css/srp_style.css

```diff
@@ -37,33 +37,51 @@ h1 {
 
 .card {
   margin-bottom: 20px;
-  padding: 24px;
+  padding: 24px 24px 24px 14px;
   box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 3px 1px -2px rgba(0, 0, 0, 0.12), 0 1px 5px 0 rgba(0, 0, 0, 0.2);
   transition: box-shadow .25s;
   border-radius: 2px;
   width: 400px;
+  display: flex;
 }
 
-.card .card-title {
+.card .content .card-title {
   font: 300 20px Roboto, Arial, sans-serif;
 }
 
-.card .card-description {
+.card .content .card-description {
   font: bold 16px Roboto, Arial, sans-serif;
   margin-top: 20px;
 }
 
-.card code {
+.card .content code {
   margin-top: 10px;
   margin-bottom: 10px;
   display: block;
 }
 
-.card p {
+.card .content p {
   margin-top: 0;
   display: block;
 }
 
+.vote {
+  width: 20px;
+  margin-right: 10px;
+  text-align: center;
+}
+
+.vote button {
+  background: none;
+  border: none;
+  outline: none;
+}
+
+.total-votes {
+  padding: 1px 6px;
```

Comment:
this is (probably) bad practice, but adding ```margin-right: 3px``` centers total votes a bit better.

Reply:
Changed around the centering with flexbox, seems to be better now